src/knowledge/utils/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .kb_utils import (
     calculate_content_hash,
     get_embedding_config,
+    merge_processing_params,
```

Comment:
这里非 bug，风格问题，这里缺少了声明
```suggestion
from .kb_utils import (
    calculate_content_hash,
    get_embedding_config,
    merge_processing_params,
    prepare_item_metadata,
    split_text_into_chunks,
    split_text_into_qa_chunks,
    validate_file_path,
)

__all__ = [
    "calculate_content_hash",
    "get_embedding_config",
    "merge_processing_params",
    "prepare_item_metadata",
    "split_text_into_chunks",
    "split_text_into_qa_chunks",
    "validate_file_path",
]

```